decidim-core/lib/decidim/features/base_controller.rb

```diff
@@ -15,7 +15,6 @@ class BaseController < Decidim::ApplicationController
       helper Decidim::ParticipatoryProcessHelper
       helper Decidim::ResourceHelper
       helper Decidim::ActionAuthorizationHelper
-      helper Decidim::MetaTagsHelper
```

Comment:
This inherits from `Decidim::ApplicationController` and that already includes the helper.